src/frontend/src/components/ChangeRequestDetailCard.tsx

```diff
@@ -3,97 +3,173 @@
+      return <ImplementedCardDetails cr={cr} />;
     default:
       return <StandardCardDetails cr={cr as StandardChangeRequest} />;
   }
 };
 
-const determineChangeRequestPillColor = (type: ChangeRequestType) => {
-  switch (type) {
-    case 'STAGE_GATE':
-      return orange[900];
-    case 'ACTIVATION':
-      return green[600];
-    case 'DEFINITION_CHANGE':
+const determineChangeRequestStatusPillColor = (status: ChangeRequestStatus) => {
+  switch (status) {
+    case 'Implemented':
       return blue[600];
-    case 'ISSUE':
+    case 'Accepted':
+      return green[600];
+    case 'Denied':
       return red[400];
+    case 'Open':
+      return purple[400];
     default:
       return grey[500];
   }
 };
 
-const StandardCardDetails = ({ cr }: { cr: StandardChangeRequest }) => {
+const determineChangeRequestTypeDesc = (type: ChangeRequestType) => {
+  switch (type) {
+    case 'ISSUE':
+      return 'Issue';
+    case 'DEFINITION_CHANGE':
+      return 'Redefinition';
+    case 'OTHER':
+      return 'Other';
+    case 'STAGE_GATE':
+      return 'Stage Gate';
+    case 'ACTIVATION':
+      return 'Activation';
+    default:
+      return 'Other';
+  }
+};
+
+const ImplementedCardDetails = ({ cr }: { cr: ChangeRequest }) => {
+  const theme = useTheme();
+  const DescriptionType = determineChangeRequestTypeDesc(cr.type);
   return (
-    <Grid container mt={1} ml={'2px'}>
-      <Grid item>
-        <Description sx={{ ml: '-4px' }} display={'inline'} />
-      </Grid>
-      <Grid item xs>
-        <Typography ml={'4px'} display={'inline'}>
-          {cr.what}
-        </Typography>
-      </Grid>
-    </Grid>
+    <Box
+      sx={{
+        backgroundColor: theme.palette.divider,
+        width: '100%',
+        minHeight: 75,
+        borderRadius: 1,
+        marginTop: 1,
+        paddingTop: 0.5,
+        paddingLeft: 1,
+        paddingRight: 1,
+        paddingBottom: 0.5
+      }}
+    >
+      <Typography variant="body1" fontSize={14} noWrap>
+        {DescriptionType + ': '}
+        <Link color="inherit" component={RouterLink} to={`${routes.PROJECTS}/${wbsPipe(cr.wbsNum)}`}>
+          {wbsPipe(cr.wbsNum)}
+        </Link>
+      </Typography>
+    </Box>
   );
 };
 
-const StageGateCardDetails = ({ cr }: { cr: StageGateChangeRequest }) => {
+const StandardCardDetails = ({ cr }: { cr: StandardChangeRequest }) => {
+  const theme = useTheme();
   return (
-    <Box ml={-1}>
-      {cr.confirmDone ? (
-        <Chip icon={<DoneAll />} label={'Done'} sx={{ backgroundColor: 'transparent' }} />
-      ) : (
-        <Chip icon={<Cancel />} label={'Not Done'} sx={{ backgroundColor: 'transparent' }} />
-      )}
+    <Box
+      sx={{
+        backgroundColor: theme.palette.divider,
+        width: '100%',
+        minHeight: 75,
+        borderRadius: 1,
+        marginTop: 1,
+        paddingTop: 0.5,
+        paddingLeft: 1,
+        paddingRight: 1,
+        paddingBottom: 0.5
+      }}
+    >
+      {cr.what}
```

Comment:
should say activate/ stage gate wbs if it's either type respectively

Reply:
oh and here and in the notes section for the implemented CR card, it should say 'No Notes' if the box is empty